modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/vulcan/openapi/contributor/RelatedObjectEntryOpenAPIContributor.java

```diff
@@ -145,14 +146,45 @@ private void _contribute(
 			_getIdParameterTemplate(schemaName), StringPool.SLASH,
 			systemObjectRelationship.getName());
 
+		Schema parameterSchema = null;
+
+		PathItem pathItem = paths.get(
+			StringBundler.concat(
+				StringPool.SLASH, version, StringPool.SLASH,
+				jaxRsApplicationDescriptor.getPath(), StringPool.SLASH,
+				_getIdParameterTemplate(schemaName)));
+
+		Operation getOperation = (pathItem != null) ? pathItem.getGet() : null;
```

Comment:
I think we usually don't use ternary operators. Please, have a look at the portal and it we don't use it, please refactor